lib/rules/selector-type-no-unknown/index.js

```diff
@@ -97,10 +74,20 @@ const rule = function (actual, options) {
             return
           }
 
+          if (
```

Comment:
Minor request... can we get this shifted to [line 61](https://github.com/stylelint/stylelint/blob/fff628f10cabf424d18c79750c7e5f71a819ad50/lib/rules/selector-type-no-unknown/index.js#L61) so the early returns are in the same alphabetical order as the options config.

Quite a lot of rules now have 5 or more `ignore/except*` options and I think there's no harm adding a bit of consistent structure to them.